### ISSUE TITLE
nn/modules/batchnorm.h redefines "F"

### ISSUE THREAD

**Issue by meganset** [pytorch/pytorch#30682]
As part of pull request #28790,
F is redefined globally in nn/modules/batchnorm.h

`namespace F = torch::nn::functional;`

F is then used once in batchnorm.h and again in instancenorm.h

not a bug, but a problem for anyone using F in some other way.
it's solvable, but I don't see this pattern anywhere else in libtorch,
was wondering if this use is an anomaly,
and if we could use

```
return torch::nn::functional::detail::batch_norm(
return torch::nn::functional::detail::instance_norm(
```
thanks

cc @yf225

**Comment by yf225** [comment]
@meganset Thanks a lot for reporting! Yes this is an anomaly, and will be fixed by https://github.com/pytorch/pytorch/pull/30684.
